### DISCUSSION TITLE
latest/version3.x/module_usage/chart_parsing

### DISCUSSION THREAD

**Discussion by giscus** [PaddlePaddle/PaddleOCR#17348] [Q&A]
# latest/version3.x/module_usage/chart_parsing

Awesome multilingual OCR toolkits based on PaddlePaddle (practical ultra lightweight OCR system, support 80+ languages recognition, provide data annotation and synthesis tools, support training and deployment among server, mobile, embedded and IoT devices)

https://www.paddleocr.ai/latest/version3.x/module_usage/chart_parsing.html

**Comment by Tomorrow0** [comment]
## PP-Chart2Table 这个模型，要什么配置呀，我是win10，安装的cpu类型，16g，i5-9400F,执行这个命令：
```bash
paddleocr chart_parsing -i "{'image': 'https://paddle-model-ecology.bj.bcebos.com/paddlex/imgs/demo_image/chart_parsing_02.png'}"
```

D:\DjangoDevelopement\ocr.venv\Lib\site-packages\paddle\tensor\creation.py:1088: UserWarning: To copy construct from a tensor, it is recommended to use sourceTensor.clone().detach(), rather than paddle.to_tensor(sourceTensor).return tensor(Setting pad_token_id to eos_token_id:151645 for open-end generation.[2025/12/14 20:18:11] paddleocr INFO: Processed item 0 in 172635.5471611023 ms 

跑了三分钟，然后执行这个

```python
from paddleocr import ChartParsing

model = ChartParsing(model_name="PP-Chart2Table", model_dir="D:\DjangoDevelopement\ocr\model\PP-Chart2Table", device="cpu")
results = model.predict(
    input={"image": r"D:\DjangoDevelopement\ocr\img\t01ccb3fbd89cfcf4ef.png"},
    batch_size=1
)
for res in results:
    res.print()
    res.save_to_json(f"./output/res.json")
```
然后日志就一直卡在
D:\DjangoDevelopement\ocr.venv\Lib\site-packages\paddle\tensor\creation.py:1088: UserWarning: To copy construct from a tensor, it is recommended to use sourceTensor.clone().detach(), rather than paddle.to_tensor(sourceTensor).return tensor(Setting pad_token_id to eos_token_id:151645 for open-end 

## 一个是需要什么样的配置才能更快的响应，能够稳定在1分钟左右，然后就是，使用哪些参数可以提升这个响应速度
